Library/Homebrew/extend/os/mac/utils/bottles.rb

```diff
@@ -6,9 +6,11 @@ class << self
       undef tag
 
       def tag
-        tag = MacOS.version.to_sym
-        tag = "#{tag}_arm".to_sym if Hardware::CPU.arm?
-        tag
+        if Hardware::CPU.intel?
+          MacOS.version.to_sym
+        else
+          "#{Hardware::CPU.arch}_#{MacOS.version.to_sym}".to_sym
```

Comment:
What's the expected string on Apple Silicon now?

Reply:
https://github.com/Homebrew/brew/blob/28f4a700acc1c5835f5565212a91b4541fcf469f/Library/Homebrew/hardware.rb#L40-L42

`arm64`